presto-orc/src/test/java/com/facebook/presto/orc/OrcTester.java

```diff
@@ -549,7 +549,7 @@ private static void assertFileContentsPresto(
                         true,
                         512 * 1024,
                         127);
-                recordReader.pushdownFilterAndProjection(options, new int[] {0}, ImmutableList.of(type));
+                recordReader.pushdownFilterAndProjection(options, new int[] {0}, ImmutableList.of(type), new Block[1]);
```

Comment:
Is it possible to use static constants for the `new int[] {0}` and `new Block[1]`? If so it would reduce the number of objects created.